client/setup.py

```diff
@@ -23,8 +23,9 @@
         "protobuf>=3.20.1",
         # gRPC,
         "grpcio>=1.46.1",
-        # PyYML package to parse yaml config files.
+        # Used to marshal config file data.
         "PyYAML>=6.0",
+        "marshmallow<=3.14.1",
```

Comment:
why the version of marshmallow needs to be less than a specific version?

Reply:
Can't upgrade beyond `3.14.1` until wee drop support for Python `3.6`.